ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/OpenSSL102/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+###################################################
+# OpenSSL v1.0.2r for building OpenJ9 on CentOS 6 #
+###################################################
+
+- name: Test if OpenSSL v1.0.2r is installed
+  shell: export LD_LIBRARY_PATH=/usr/local/openssl-1.0.2/lib &&  /usr/local/openssl-1.0.2/bin/openssl version | awk '{print $2}' | cut -c 1-5
+  when:
+    - (ansible_distribution == "RedHat" or ansible_distribution == "CentOS")
+    - ansible_distribution_major_version == "6" 
+  register: openssl_version
+  tags: openssl
+
```

Comment:
Tags should be modified such that either/or openssl role can be targeted/omitted.